athina/steps/code_execution_v2.py

```
@@ -396,7 +399,8 @@
         """
         print_output = str()
         try:
-            self._create_or_initialize_sandbox()
+            session_id = input_data.get("session_id", None)
```

Ruff (0.8.2)

Use `input_data.get("session_id")` instead of `input_data.get("session_id", None)`

Replace `input_data.get("session_id", None)` with `input_data.get("session_id")`

(SIM910)